baseplate/sidecars/live_data_watcher.py

```diff
@@ -128,25 +90,76 @@ def on_change(self, data: Optional[bytes], _znode_stat: ZnodeStat) -> None:
             if self.owner and self.group:
                 os.fchown(tmpfile.fileno(), self.owner, self.group)
             os.fchmod(tmpfile.fileno(), self.mode)
-            try:
-                json_data = json.loads(data.decode("UTF-8"))
-            except json.decoder.JSONDecodeError:
-                # If JSON fails to decode, still write the bytes data since
-                # we don't necessarily know if the the contents of the znode
-                # had to be valid JSON anyways.
-                tmpfile.write(data)
-            else:
-                # If no exceptions, we have valid JSON, and can parse accordingly.
-                data = NodeWatcher.get_data_to_write(json_data, data)
-                if data is None:
-                    logger.warning(
-                        "No data written to destination file. Something is likely misconfigured."
-                    )
-                    return
-                tmpfile.write(data)
+
+            tmpfile.write(data)
         os.rename(self.dest + ".tmp", self.dest)
 
 
+def _parse_loader_type(data: bytes) -> LoaderType:
+    try:
+        json_data = json.loads(data.decode("UTF-8"))
+    except (UnicodeDecodeError, json.decoder.JSONDecodeError):
+        logger.debug("Data is not parseable as JSON, loading as PASSTHROUGH")
```

Comment:
Do we know how often this log message might happen?

Reply:
![Screen Shot 2022-06-02 at 11 13 16 AM](https://user-images.githubusercontent.com/1282393/171662219-3b7d67d0-b18a-42f7-b37a-134cd4c60dea.png)

Basically if the zk node is empty or contains a string that isn't wrapped in quotes